orttraining/orttraining/python/training/ortmodule/_graph_execution_manager.py

```diff
@@ -384,6 +398,17 @@ def _get_exported_model(self, input_schema: _ModelInputOutputSchemaType, *inputs
                 exported_model, self._runtime_options.enable_custom_autograd_function
             )
 
+            # Cache model for future runs
+            if cache_dir:
+                self._logger.info("ORTModule cache optimization is ON.")
+                if not os.path.exists(cache_dir):
+                    os.makedirs(cache_dir, exist_ok=True)
+                filename = os.path.join(
+                    cache_dir, f"{hash_fn(str(self._flattened_module).encode()).hexdigest()}_{rank}.onnx"
+                )
+                self._logger.info(f"Caching model for future runs to {filename}.")
```

Comment:
it will be good to add something like this
```suggestion
                self._logger.info(f"Caching model for future runs to {filename}. Set environment variable ORT_CACHE_DIR={cache_dir} to load the cached model on next run.")
```

Reply:
@askhade the user does not need to set this environment variable again to load the cached model on the next run.

From the user perspective, when they set ORTMODULE_CACHE_DIR the first time, they'll see a log that the optimization is ON (L403) and that the model is being cached (L409). Then, since ORTMODULE_CACHE_DIR is already set, on the next run they'll again see a log that the optimization is ON (L329) and another log that the code has automatically pick up the cached model from ORTMODULE_CACHE_DIR along with instructions on how to reset the cache by deleting the relevant file (L334).